perception_launch/launch/object_recognition/detection/lidar_based_detection.launch.xml

```diff
@@ -100,7 +101,7 @@
     <include file="$(find-pkg-share object_merger)/launch/object_association_merger.launch.xml">
       <arg name="input/object0" value="temporary_merged_objects"/>
       <arg name="input/object1" value="detection_by_tracker/objects"/>
-      <arg name="output/object" value="objects"/>
+      <arg name="output/objects" value="$(var output/objects)"/>
```

Comment:
is it correct?
if you confirmed to work, it's ok

Reply:
fixed